balancer/balancer.go

```diff
@@ -167,6 +167,16 @@ type ClientConn interface {
 	//
 	// Deprecated: Use the Target field in the BuildOptions instead.
 	Target() string
+
+	// MetricsRecorder provides the metrics recorder that balancers can use to
+	// record metrics. Balancer implementations which do not register metrics on
+	// metrics registry and record on them can ignore this method.
```

Comment:
How about also adding: `Guaranteed to never be nil` or `Must always be safe to call` or something?

Reply:
Added a line.